assignments/objects.js

```diff
@@ -17,28 +17,53 @@ const example = {
   "gender": "F"
 }
 
+//constructor notation 
+function Person(id, email, name, gender){
+  this.id = id;
+  this.name = name;
+  this.email = email;
+  this.gender = gender;
+}
+
+
 // Write your intern objects here:
+const mitzi = new Person(1,'mmelloy0@psu.edu','Mitzi','F');
```

Comment:
Yep, your way of doing this certainly requires less lines of code and is more concise.